nemo/nemo/backends/pytorch/actions.py

```diff
@@ -997,6 +1000,132 @@ def _get_all_modules(
                     for module in callchain:
                         self.modules.add(module[0])
 
+    @staticmethod
+    def __module_export(module,
+                        output,
+                        d_format: DeploymentFormat,
+                        input_example=None,
+                        output_example=None):
+        # Check if output already exists
+        destination = Path(output)
+        if destination.exists():
+            raise FileExistsError(f"Destination {output} already exists. "
+                                  f"Aborting export.")
+
+        input_names = list(module.input_ports.keys())
+        output_names = list(module.output_ports.keys())
+        dynamic_axes = {}
+
+        def __extract_dynamic_axes(port_name: str, ntype: NeuralType,
+                                   dynamic_axes: Dict):
+            if ntype.axis2type is not None and ntype.axis2type != {}:
```

Comment:
I think `if ntype.axis2type:` should take care of both cases (None and empty dict => False)

Reply:
done